WebApi/Controllers/WorkplacesController.cs

```diff
@@ -14,13 +14,31 @@ namespace WebApi.Controllers
     public class WorkplacesController : BaseApiController
     {
 
+
         [HttpGet]
-        public async Task<IActionResult> GetAll([FromQuery]GetWorkplaceListQueryRequest request)
+        public async Task<IActionResult> GetAll(
+            [FromQuery] string mapId,
```

Comment:
why have you changed it ?

Reply:
Because we should hide some fields of request like RecurringDay, startdate and EndDate